packages/components/src/form-token-field/style.scss

```
@@ -181,4 +181,14 @@
 		background: $components-color-accent;
 		color: $white;
 	}
+
+	&[aria-disabled="true"] {
+		pointer-events: none;
+		color: $gray-600;
+
+		&.is-selected {
+			color: $gray-100; // fallback
+			color: rgba( 255, 255, 255, 0.5 );
```

this line triggers lint-style error:

```
✖ wp-scripts lint-style found some errors. Please fix them and try committing again.
 
packages/components/src/form-token-field/style.scss
191:16  ✖  Unexpected whitespace after "("   function-parentheses-space-inside
191:35  ✖  Unexpected whitespace before ")"  function-parentheses-space-inside
```

I committed this change with force

---

Yes, the usual extra spacing based on WP coding style doesn't really apply to CSS functions. Perhaps we can just satisfy the linter?

```suggestion
			color: rgba(255, 255, 255, 0.5);
```